watertap/data/techno_economic/energy_recovery.yaml

```diff
@@ -2,29 +2,15 @@ default:
   energy_electric_flow_vol_inlet:
     value: -3.17
     units: kWh/m^3
-  specific_capital_cost_flow_vol_inlet:
-    flow_basis_par:
+  capital_cost:
+    basis: flow_vol
+    cost_factor: None
+    reference_state:
       value: 3.943
       units: m^3/hr
-    cap_basis_par:
-      value: 0.1
-      units: M$
-    cap_exp_par:
+    capital_a_parameter:
+      value: 1e5
+      units: USD_2020
```

Comment:
Check the year for this - below it says 2020.

Reply:
I set the unit as USD_2020 as indicated in the original yaml file. What is the problem here?

Reply:
I have no idea a this point. For some reason when I first read this I thought the year here was different.